app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -1,22 +1,22 @@
-<p>Welcome, <%= @resource.name %>!</p>
-
 <% account_user = @resource&.account_users&.first %>
-<% if account_user&.inviter.present? && @resource.unconfirmed_email.blank? %>
-  <p><%= account_user.inviter.name %>, with <%= account_user.account.name %>, has invited you to try out <%= global_config['BRAND_NAME'] || 'Chatwoot' %>! </p>
-<% end %>
-
-<% if @resource.confirmed? %>
-<p>You can login to your account through the link below:</p>
-<% else %>
-<p>You can confirm your account email through the link below:</p>
-<% end %>
 
 <% if @resource.unconfirmed_email.present? %>
-<p><%= link_to 'Confirm my account', frontend_url('auth/confirmation', confirmation_token: @token) %></p>
+  <p>
+    Hi <%= @resource.name %>, and welcome to <%= global_config['BRAND_NAME'] || 'Chatwoot' %>! We have a lot of awesome stuff ready for you on the next screen. 
```

Comment:
"We have a lot of awesome stuff ready for you on the next screen." -> we could improve this copy. 

"We have a suite of powerful tools ready for you to explore. Before that we quickly need to verify..."